baseplate/experiments/__init__.py

```diff
@@ -77,6 +78,16 @@ def _get_config(self, name):
             logger.warning("Could not load experiment config: %s", str(exc))
             return
 
+    def _get_experiment(self, name):
+        if name not in self._experiment_cache:
```

Comment:
> My thinking was you get a new Experiments object for each request so you're only caching away the file watcher for a single request.

This is probably safe for now, but when we have long running processes (crons, queue consumers) in baseplate services this will mess them up.

If we have spans around this stuff we can get a good idea of the real life cost of excessive/unbatched feature checks.